pkg/tarmak/tarmak.go

```diff
@@ -144,24 +144,33 @@ func (t *Tarmak) initializeModules() {
 func (t *Tarmak) initializeConfig() error {
 	var err error
 
+	environmentDestroy := false
+	if t.flags.Environment.Destroy.Name != "" {
```

Comment:
We should have a better way to initialize the config. I think we should set the current env to the one to destroy then run this func again

Reply:
I don't really like the fact that we are using Name here in our API which is not being used on the CLI. I think it should be possible to set the current environment, `t.config.CurrentCluster = fmt.Sprintf("%s-", args[0])`
then call `t.initializeConfig()` again within `cmd.DestroyEnvrionment()`